client/src/components/BookingList/Booking.tsx

```diff
@@ -0,0 +1,39 @@
+import { Booking as IBooking } from '../../interface/Booking';
+import { Avatar, Card, Grid, IconButton, Typography, Box } from '@mui/material';
+import SettingsIcon from '@mui/icons-material/Settings';
+import format from 'date-fns/format';
+
+interface PropTypes {
+  booking: IBooking;
```

Comment:
```suggestion
  booking: Booking;
```
We don't need to prefix our interfaces with 'I' in JS. Ditto where applicable!